packages/server/src/lib/User.ts

```diff
@@ -75,9 +77,46 @@ async function loginUser(
   return resUser;
 }
 
-export async function getLoggedInUser(userJwtPayload: UserJWTPayload | null | undefined): Promise<UserMeResponse> {
+async function getUserByUsernameOrId(
+  payload: UserGetPayload
```

Comment:
as the name of the function suggests it should take in an argument of Username or Id
```suggestion
  usernameOrId: string
```

Reply:
![image](https://user-images.githubusercontent.com/37513070/172602109-3050bbd8-880d-4bb9-b855-70ba51399763.png)

Reply:
Done